test/gurka/gurka.h

```diff
@@ -515,27 +515,6 @@ inline void build_pbf(const nodelayout& node_locations,
 
 } // namespace detail
 
-/**
- * Generate a new GraphReader that doesn't re-use a previously
- * statically initizalized tile_extract member variable.
- *
- * Useful if you need to reload a tile extract within the same
- * process
- */
-std::shared_ptr<valhalla::baldr::GraphReader>
-make_clean_graphreader(const boost::property_tree::ptree& pt) {
```

Comment:
I'm moving this into the new `test/util/traffic_utils.h` file as it's useful outside of gurka tests.